### DISCUSSION TITLE
Login/Signup

### DISCUSSION THREAD

**Discussion by flip111** [flora-pm/flora-server#520] [Feature Requests]
1.
The url could be simpler `https://flora.pm/sessions/new` to `https://flora.pm/login`

2.
There is no signup options. As long as this option is not there rename the navigation link to `Login`.

3.
Some explanation for the purpose of this page would be nice. Something like:

This is the login page for beta testing features, we intend to make new registrations available when <condition>. Planned features are X, Y, Z. Sign up for the mailing list if you want to be notified when we open registration.

**Comment by tchoutri** [comment]
Number 2 has been addressed, Number 1 won't be yet. Number 3 won't be addressed because there are no plans for opening sign-ups.
